docs/spmd.md

```diff
@@ -552,3 +552,16 @@ the XLA-based auto-sharding pass:
 - `XLA_AUTO_SPMD_MESH`: logical mesh shape to be used for auto-sharding. For example,
 `XLA_AUTO_SPMD_MESH=2,2` corresponds to a 2-by-2 mesh with 4 global devices. If unset,
 a default device mesh shape of `num_devices,1` will be used.
+
+### Activation Sharding
+
+In the 2.3 release, PyTorch/XLA added the custom op `dynamo_mark_sharding` which can be used to perform the activation sharding in a `torch.compile` region. This is part of our ongoing effort to make `torch.compile` + `GSPMD` to be the recommended way of doing the model inference using PyTorch/XLA. Example of using this custom op:
+```
+# Activation output sharding
+import torch_xla.experimental.dynamo_mark_sharding
+device_ids = [i for i in range(self.num_devices)] # List[int]
+mesh_shape = [self.num_devices//2, 1, 2] # List[int]
+axis_names = 'None' # string version of axis_names
```

Comment:
can we use `axis_names ` instead of numbers for `partition_spec` ? I want to update this doc to encourage user to give each mesh axis a name. I think this is more clear.

Reply:
Sg, just updated